packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -932,10 +932,12 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
       }
       final RenderBox childRenderBox = _childKey.currentContext!.findRenderObject()! as RenderBox;
       final Size childSize = childRenderBox.size;
```

Comment:
This variable childSize would no longer be used with these changes and could be removed.